material-sheet-fab/build.gradle

```diff
@@ -29,7 +29,7 @@ android {
     buildToolsVersion "23.0.3"
 
     defaultConfig {
-        minSdkVersion 14
+        minSdkVersion 15
```

Comment:
Not anymore. It's now supporting SDK >= 15:
https://github.com/ozodrukh/CircularReveal